specs/src/networking.md

```diff
@@ -21,7 +21,10 @@
 Defined as `ConsensusProposal`:
 
 ```protobuf
-{{#include ./proto/consensus.proto:ConsensusProposal}}
+// ConsensusProposal message definition (removed from specs)
+message ConsensusProposal {
```

Comment:
This protobuf section isn't helpful and I don't think Consensus Proposal is a real type in celestia-core or celestia-app so IMO we should remove it.